homeassistant/components/advantage_air/__init__.py

```diff
@@ -0,0 +1,92 @@
+"""Advantage Air climate integration."""
+
+import collections.abc
+from datetime import timedelta
+import logging
+
+from advantage_air import ApiError, advantage_air
+
+from homeassistant.const import CONF_IP_ADDRESS, CONF_PORT
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import ADVANTAGE_AIR_RETRY, DOMAIN
+
+ADVANTAGE_AIR_SYNC_INTERVAL = 15
+ADVANTAGE_AIR_PLATFORMS = ["climate", "cover"]
```

Comment:
Limit the PR to introducing one platform

Reply:
See https://github.com/home-assistant/core/pull/40159